converters/convertGBP.js

```diff
@@ -0,0 +1,22 @@
+
+// Exchange rates
+const RATES = {
+    USD: 1.40, 
+    EUR: 1.15,
+    RUB: 104.27
+};
+
+// Conversion function
+function convertToGBP(value, fromCurrency) {
```

Comment:
good name for function

Reply:
tyvm that means a lot to me!!!!!!!111!!

Reply:
I hope you have some more meaningful conversations guys! 😄 @Scott-Crawley @umaimahyusuf